float64[] origin
float64[] head
float64[] leftShoulder
float64[] rightShoulder
float64[] hipTorso
string timestamp
